### PR TITLE
Attempt to fix Issue #55184 for Hacktoberfest

### PR DESCRIPTION
**Pull request by bobfuru**
Not sure if this is what was intended with the [comment](https://github.com/openshift/openshift-docs/issues/55184#issuecomment-1718437073) but thought I'd attemp to address it for H-fest. Thanks!

### CODE REVIEW

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Comment by bergerhoffer** [comment]
I marked this hacktoberfest-accepted since we got delayed in reviewing due to GA. 

But it looks like {ocp-version} doesn't resolve. @rolfedh you suggested this [in this comment](https://github.com/openshift/openshift-docs/issues/55184#issuecomment-1718437073). Do you know if that's supposed to work? @bmcelvee @EricPonvelle is there an attribute for OCP version? Or what should be used in this link?

**Comment by rolfedh** [comment]
My mistake. `:ocp-version:` exists only in `openshift-docs/_attributes/attributes-microshift.adoc`. I don't see an equivalent in `_attributes/common-attributes.adoc`.

File: modules/osd-intro.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [id="osd-intro_{context}"]
 = An overview of {product-title}
 
-{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
+{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```
Comment (rolfedh) [inline modules/osd-intro.adoc:8]: I probably should have recommended 
```suggestion
{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed xref:../architecture/control-plane.adoc#control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```

File: modules/osd-intro.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 * Open source development model. Development is completed in the open, and the source code is available from public software repositories. This open collaboration fosters rapid innovation and development.
 
-To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see link:https://docs.openshift.com/container-platform/4.7/architecture/understanding-development.html[Understanding {OCP} development] in the {OCP} documentation.
+To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see link:https://docs.openshift.com/container-platform/{ocp-version}/architecture/understanding-development.html[Understanding {OCP} development] in the {OCP} documentation.
```
Comment (rolfedh) [inline modules/osd-intro.adoc:18]:
```suggestion
To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see xref:../architecture/understanding-development.adoc#understanding-development[Understanding {OCP} development].
```

**Comment by rolfedh** [comment]
Hi @bobfuru. Yesterday, I mistakenly suggested adding an xref to a module, which isn't allowed. I'm discussing the best approach with the OpenShift docs team and will get back to you.

File: modules/osd-intro.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 [id="osd-intro_{context}"]
 = An overview of {product-title}
 
-{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
+{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed xref:../architecture/control-plane.adoc#control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```
Comment (rolfedh) [inline modules/osd-intro.adoc:8]:
```suggestion
{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```

File: modules/osd-intro.adoc [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 * Open source development model. Development is completed in the open, and the source code is available from public software repositories. This open collaboration fosters rapid innovation and development.
 
-To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see link:https://docs.openshift.com/container-platform/4.7/architecture/understanding-development.html[Understanding {OCP} development] in the {OCP} documentation.
+To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see xref:../architecture/understanding-development.adoc#understanding-development[Understanding {OCP} development].
```
Comment (rolfedh) [inline modules/osd-intro.adoc:18]:
```suggestion
To learn about options for assets you can create when you build and deploy containerized Kubernetes applications in {OCP}, see link:https://docs.openshift.com/container-platform/{ocp-version}/architecture/understanding-development.html[Understanding {OCP} development].
```

**Review by rolfedh — COMMENTED** [review]
Okay, @bobfuru. Thanks for your patience while I sorted this one out. After a conversation with @bmcelvee, we came around to restoring the `link:` with a URL **and adding an attribute for the version number**. 

I'd also like to ask @kalexand-rh to have a look at this before you go ahead with the following bit. If ~she~ they agree, would you do the following: 

Would you add `:ocp-version: 4.14` to the `_attributes/common-attributes.adoc` file? 

Thanks.

**Comment by bobfuru** [comment]
> Okay, @bobfuru. Thanks for your patience while I sorted this one out. After a conversation with @bmcelvee, we came around to restoring the `link:` with a URL **and adding an attribute for the version number**.
> 
> I'd also like to ask @kalexand-rh to have a look at this before you go ahead with the following bit. If she agrees, would you do the following:
> 
> Would you add `:ocp-version: 4.14` to the `_attributes/common-attributes.adoc` file?
> 
> Thanks.

I'd like to gracefully bow out at this point and ask that you create a new PR to address the issue. I've kind of lost track of the conversation and feel you've probably got a better handle on how to proceed than me. Is that okay with you, @rolfedh?

**Comment by rolfedh** [comment]
I understand @bobfuru. I'll take it from here. Thanks!

**Comment by rolfedh** [comment]
See https://github.com/openshift/openshift-docs/pull/67265 instead

**Comment by kalexand-rh** [comment]
> > Okay, @bobfuru. Thanks for your patience while I sorted this one out. After a conversation with @bmcelvee, we came around to restoring the `link:` with a URL **and adding an attribute for the version number**.
> > I'd also like to ask @kalexand-rh to have a look at this before you go ahead with the following bit. If she agrees, would you do the following:
> > Would you add `:ocp-version: 4.14` to the `_attributes/common-attributes.adoc` file?
> > Thanks.
> 
> I'd like to gracefully bow out at this point and ask that you create a new PR to address the issue. I've kind of lost track of the conversation and feel you've probably got a better handle on how to proceed than me. Is that okay with you, @rolfedh?

@rolfedh, I don't use she/her pronouns.
@bmcelvee, do you have a plan to maintain this attribute? Since it's automatically provided for OCP through asciibinder, there's a huge risk that it's going to go stale unless you intentionally update it. I am neutral on adding it if SD is going to maintain it and opposed otherwise.
@bobfuru, thank you for opening the original PR!

**Comment by rolfedh** [comment]
I apologize, @kalexand-rh. Thanks for letting me know. In general, I'm moving toward using non-gendered pronouns or avoiding personal pronouns altogether to avoid such errors.

**Pull request closed without merging**